packages/blocks/styles/components/t-badge.css

```diff
@@ -33,26 +33,35 @@
 /* Badge skins */
 .t-badge-default {
     @apply bg-gray-100
+           border-gray-200
            mix-blend-multiply
            text-gray-500;
 }
 
 .t-badge-primary {
     @apply bg-blue-600
+           border-blue-700
            text-white;
 }
 
 .t-badge-danger {
     @apply bg-red-600
+           border-red-700
            text-white;
 }
 
 .t-badge-warning {
     @apply bg-yellow-500
+           border-yellow-600
            text-white;
 }
 
 .t-badge-success {
     @apply bg-green-600
+           border-green-700
            text-white;
 }
+
+.t-badge-rounded-full {
+  @apply rounded-full;
+}
```

Comment:
I think we can use `rounded-full` directly.
Do you have any reason for this re-definition? @emilyvandewalle

Reply:
I see, it's a bit redundant indeed, if there would be additional styles specific to rounded badges (for example more horizontal spacing) then it could make sense to have a separate class, but for now it's not needed indeed. I'll remove it 👍